sdk/workloadssapvirtualinstance/azure-resourcemanager-workloadssapvirtualinstance/src/main/java/com/azure/resourcemanager/workloadssapvirtualinstance/models/ManagedServiceIdentityType.java

```diff
@@ -1,62 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-// Code generated by Microsoft (R) TypeSpec Code Generator.
-
-package com.azure.resourcemanager.workloadssapvirtualinstance.models;
-
-import com.azure.core.util.ExpandableStringEnum;
-import java.util.Collection;
-
-/**
- * Type of managed service identity (where both SystemAssigned and UserAssigned types are allowed).
- */
-public final class ManagedServiceIdentityType extends ExpandableStringEnum<ManagedServiceIdentityType> {
```

Comment:
Removing this type is a breaking change (if this library has released a GA).

Reply:
Service has done some breaks after this commit.
https://github.com/Azure/azure-rest-api-specs/pull/31125
https://github.com/Azure/azure-rest-api-specs/pull/31675
(now service is pretty easy to merge breaks into specs repo -- they just add the "BreakingChange-Approved-BugFix" label themselves)

The class itself seems changed to `SAPVirtualInstanceIdentity`. We might be able to clientName it in next release.

However, the overall breaks may not be absolved in SDK.